inputs/complex-list-item.vue

```diff
@@ -33,13 +33,17 @@
   }
 
   .complex-list-item-actions {
-    margin-top: 10px;
-    width: 100%;
+    border-top: 1px solid $card-border-color;
+    margin: 16px -16px 0;
+    // account for outer padding
+    padding: 8px 16px;
+    width: calc(100% + 32px);
```

Comment:
Could this be handled instead with `box-sizing: border-box`?

Reply:
It's already using border-box, I believe